docs/dev_guide/diagnostic_ref/dpct1051.rst

```diff
@@ -27,4 +27,40 @@ Suggestions to Fix
 
 Review the logic and adjust it manually.
 
-.. include:: /_include_files/support-msg.txt
\ No newline at end of file
+.. include:: /_include_files/support-msg.txt
+
+For example, this original CUDA\* code:

```

Comment:
Remove "*" from CUDA - already attributed earlier on page. Need to attribute first use only.

Reply:
done

Reply:
done